rules/integrations/aws/privilege_escalation_aws_iam_administratoraccess_policy_attached_to_new_user.toml

```diff
@@ -0,0 +1,152 @@
+
+### Response and remediation
+
+- Initiate the incident response process based on the outcome of the triage.
+- Disable or limit the account during the investigation and response.
+    - Rotate user credentials
+    - Remove the `AdministratorAccess` policy from the affected user(s)
+- Identify the possible impact of the incident and prioritize accordingly; the following actions can help you gain context:
+    - Identify the account role in the cloud environment.
+    - Assess the criticality of affected services and servers.
+    - Work with your IT team to identify and minimize the impact on users.
+    - Identify if the attacker is moving laterally and compromising other accounts, servers, or services.
+    - Identify any regulatory or legal ramifications related to this activity.
+- Investigate credential exposure on systems compromised or used by the attacker to ensure all compromised accounts are identified. 
+    - Rotate secrets or delete API keys as needed to revoke the attacker's access to the environment. 
+    - Work with your IT teams to minimize the impact on business operations during these actions.
+- Remove unauthorized new accounts, and request password resets for other IAM users.
+- Consider enabling multi-factor authentication for users.
+- Review the permissions assigned to the implicated user to ensure that the least privilege principle is being followed.
+- Implement security best practices [outlined](https://aws.amazon.com/premiumsupport/knowledge-center/security-best-practices/) by AWS.
+- Take the actions needed to return affected systems, data, or services to their normal operational levels.
+- Identify the initial vector abused by the attacker and take action to prevent reinfection via the same vector.
+- Using the incident response data, update logging and audit policies to improve the mean time to detect (MTTD) and the mean time to respond (MTTR).
+"""
+references = [
+    "https://docs.aws.amazon.com/IAM/latest/APIReference/API_AttachUserPolicy.html",
+    "https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AdministratorAccess.html",
+    "https://docs.aws.amazon.com/IAM/latest/APIReference/API_CreateUser.html",
+]
+risk_score = 47
+rule_id = "2c0e450c-527e-4149-b810-4fc9ab29ae65"
+severity = "medium"
+tags = [
+    "Domain: Cloud",
+    "Data Source: AWS",
+    "Data Source: Amazon Web Services",
+    "Data Source: AWS IAM",
+    "Use Case: Identity and Access Audit",
+    "Tactic: Privilege Escalation",
+    "Tactic: Persistence",
+    "Resources: Investigation Guide",
+    "Rule Type: Higher-Order Rule",
+]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+signal where kibana.alert.rule.rule_id in ("2fdc2e81-4b8d-405a-972d-4351fcf23b4a", "58febb7b-f589-43b3-9110-14756abe0aad")
```

Comment:
If this is an `OR` statement then we are alerting if any of the BBR rules referenced match? If so, this seems to be a promotional rule of sorts.

Reply:
So then you think we should have 2 rules that share the same goal? I did it this way thinking it'd be easier to manage and keep track of alerts for this one rule.